src/main/java/uk/gov/laa/ccms/caab/bean/validators/client/ClientAddressSearchValidator.java

```diff
@@ -1,37 +1,43 @@
 package uk.gov.laa.ccms.caab.bean.validators.client;
 
 import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
 import org.springframework.validation.Errors;
-import uk.gov.laa.ccms.caab.bean.ClientDetails;
+import org.springframework.validation.Validator;
+import uk.gov.laa.ccms.caab.bean.ClientFormDataAddressSearch;
 
 /**
  * Validates the uprn details provided by client address search.
  */
 @Component
-public class ClientAddressSearchValidator extends AbstractClientAddressValidator {
+public class ClientAddressSearchValidator implements Validator {
```

Comment:
Should this not extend AbstractValidator (or another) like the others?

Reply:
It does not need access to any of the methods inside the abstract validator so it can inherit from the default spring one